runelite-client/src/main/java/net/runelite/client/plugins/profiles/ProfilePanel.java

```diff
@@ -0,0 +1,167 @@
+
+@Slf4j
+class ProfilePanel extends JPanel
+{
+	private static final ImageIcon DELETE_ICON;
+	private static final ImageIcon DELETE_HOVER_ICON;
+
+	static
+	{
+		final BufferedImage deleteImg = ImageUtil.getResourceStreamFromClass(ProfilesPlugin.class, "delete_icon.png");
+		DELETE_ICON = new ImageIcon(deleteImg);
+		DELETE_HOVER_ICON = new ImageIcon(ImageUtil.alphaOffset(deleteImg, -100));
+	}
+
+	private final String loginText;
+
+	ProfilePanel(final Client client, String data, ProfilesConfig config)
+	{
+		String[] parts = data.split(":");
+		this.loginText = parts[1];
+
+		final ProfilePanel panel = this;
+
+		setLayout(new BorderLayout());
+		setBackground(ColorScheme.DARKER_GRAY_COLOR);
+
+		JPanel labelWrapper = new JPanel(new BorderLayout());
+		labelWrapper.setBackground(ColorScheme.DARKER_GRAY_COLOR);
+		labelWrapper.setBorder(new CompoundBorder(
+			BorderFactory.createMatteBorder(0, 0, 1, 0, ColorScheme.DARK_GRAY_COLOR),
+			BorderFactory.createLineBorder(ColorScheme.DARKER_GRAY_COLOR)
+		));
+
+		JPanel panelActions = new JPanel(new BorderLayout(3, 0));
+		panelActions.setBorder(new EmptyBorder(0, 0, 0, 8));
+		panelActions.setBackground(ColorScheme.DARKER_GRAY_COLOR);
+
+		JLabel delete = new JLabel();
+		delete.setIcon(DELETE_ICON);
+		delete.setToolTipText("Delete account profile");
+		delete.addMouseListener(new MouseAdapter()
+		{
+			@Override
+			public void mousePressed(MouseEvent e)
+			{
+				panel.getParent().remove(panel);
+				ProfilesPanel.removeProfile(String.join(":", parts));
+			}
+
+			@Override
+			public void mouseEntered(MouseEvent e)
+			{
+				delete.setIcon(DELETE_HOVER_ICON);
+			}
+
+			@Override
+			public void mouseExited(MouseEvent e)
+			{
+				delete.setIcon(DELETE_ICON);
+			}
+		});
+
+		panelActions.add(delete, BorderLayout.EAST);
+
+		JLabel label = new JLabel();
+		label.setText(parts[0]);
+		label.setBorder(null);
+		label.setBackground(ColorScheme.DARKER_GRAY_COLOR);
+		label.setPreferredSize(new Dimension(0, 24));
+		label.setForeground(Color.WHITE);
+		label.setBorder(new EmptyBorder(0, 8, 0, 0));
+
+		labelWrapper.add(label, BorderLayout.CENTER);
+		labelWrapper.add(panelActions, BorderLayout.EAST);
+		label.addMouseListener(new MouseAdapter()
+		{
+			@Override
+			public void mousePressed(MouseEvent e)
+			{
+				log.info("label", e);
+				if (SwingUtilities.isLeftMouseButton(e))
+				{
+					if (client.getGameState() == GameState.LOGIN_SCREEN)
+					{
+						client.setUsername(loginText);
+					}
+				}
+			}
+		});
+
+		JPanel bottomContainer = new JPanel(new BorderLayout());
+		bottomContainer.setBorder(new EmptyBorder(8, 0, 8, 0));
+		bottomContainer.setBackground(ColorScheme.DARKER_GRAY_COLOR);
+		bottomContainer.addMouseListener(new MouseAdapter()
+		{
+			@Override
+			public void mousePressed(MouseEvent e)
+			{
+				log.info("profilePanel", e);
+				if (SwingUtilities.isLeftMouseButton(e))
+				{
+					if (client.getGameState() == GameState.LOGIN_SCREEN)
```

Comment:
See above comment

Reply:
In the case of enums, `.equals()` just uses `==` so it makes absolutely no difference.